src/olympia/addons/models.py

```diff
@@ -515,7 +516,7 @@ def delete(self, msg='', reason=''):
             # The last parameter is needed to automagically create an AddonLog.
             amo.log(amo.LOG.DELETE_ADDON, self.pk, unicode(self.guid), self)
             self.update(status=amo.STATUS_DELETED, slug=None,
-                        _current_version=None)
+                        _current_version=None, modified=date.now())
```

Comment:
I was surprised we need to do this, but I guess it's because our `.update()` method doesn't do a real `save()` so I guess it prevents `auto_now` from working correctly.